src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+  const [showError, setShowError] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isShowError, setIsShowError] = useState<boolean>(false);
```
[Tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
Fix all cases if present